tests/tck/conftest.py

```diff
@@ -562,6 +578,19 @@ def result_should_contain(request, result, graph_spaces):
     )
 
 
+@then(parse("the result should contain, replace the holders with cluster info:\n{result}"))
```

Comment:
I think you extend it to more general, not just cluster info but anything could eval by python runtime.

Reply:
agreed, but I'm not sure what else should be add except cluster info.
we could change it in future if we have such cases?

Reply:
ok